### DISCUSSION TITLE
Red Warning Color

### DISCUSSION THREAD

**Discussion by msenol86** [microsoft/pylance-release#1994] [Q&A] [answered]
![Screen Shot 2021-10-27 at 23 20 21](https://user-images.githubusercontent.com/2547551/139140971-7ad0dbf8-7835-4665-b612-9d3cb5f5ef3a.png)

VSCode shows linting warning (too many blank lines) with red color? How can I change that? I want to make it something like yellow, orange or blue

I have tried [this](https://stackoverflow.com/a/45899100) but no luck

**Comment by jakebailey** [comment, marked as answer]
If you're getting a message "too many blank lines", I don't think that's from Pylance; we don't do style checks like that. Maybe it's pylint or some other tool you have installed?

**Reply by msenol86** [reply]
ok thx a lot 👍
